Dockerfile

```diff
@@ -18,12 +18,35 @@ RUN /opt/miniconda/bin/conda init bash
 # Download and extract the pyannote speaker diarization data
 RUN wget -O - https://minutesgeneratorpublic.s3.amazonaws.com/data-2023-03-25-02.tar.gz | tar xz -C /
 
+# Copy the environment YAML file
 COPY server_env.yml /tmp/server_env.yml
 
 # Copy the setup script into the image
 COPY setup.sh /usr/local/bin/setup.sh
 
-# Make the setup script executable
-RUN chmod +x /usr/local/bin/setup.sh
+# Make the setup script executable and run it
+RUN chmod +x /usr/local/bin/setup.sh && \
+    /usr/local/bin/setup.sh
```

Comment:
We talked offline that theres limited disk space on the runpod vps, so best not to bundle the python packages into the image that setup.sh installs